packages/common/client-utils/src/test/types/validateClientUtilsPrevious.generated.ts

```diff
@@ -24,26 +24,26 @@ type TypeOnly<T> = T extends number
 /*
 * Validate forward compat by using old type in place of current type
 * If breaking change required, add in package.json under typeValidation.broken:
-* "ClassDeclaration_Buffer": {"forwardCompat": false}
+* "RemovedClassDeclaration_Buffer": {"forwardCompat": false}
```

Comment:
`Buffer` in `bufferNode.ts` is an `export declare class`, just a declaration that the class exists somewhere (in this case, Node provides it), so it shouldn't be exported from our package.

Reply:
Actually, seems like the solution is to export it as a type only.

Trying to export it without any modifiers runs into issues like this one, when running `npm run test:mocha` in the table-document package:

```console
npm run test:mocha

> @fluid-example/table-document@2.0.0-rc.1.0.0 test:mocha
> mocha --config src/test/.mocharc.js dist/test

Writing test results relative to package to nyc/local-junit-report.xml and nyc/local-junit-report.json

file:///workspaces/FluidFramework/packages/common/client-utils/lib/indexNode.mjs:5
export { Buffer, bufferToString, IsoBuffer, stringToBuffer, Uint8ArrayToString, } from "./bufferNode.mjs";
         ^^^^^^
SyntaxError: The requested module './bufferNode.mjs' does not provide an export named 'Buffer'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at ModuleJob.run (node:internal/modules/esm/module_job:191:5)
    at ModuleLoader.import (node:internal/modules/esm/loader:336:24)
    at importModuleDynamicallyWrapper (node:internal/vm/module:429:15)
    at formattedImport (/workspaces/FluidFramework/node_modules/.pnpm/mocha@10.2.0/node_modules/mocha/lib/nodejs/esm-utils.js:9:14)
    at Object.exports.loadFilesAsync (/workspaces/FluidFramework/node_modules/.pnpm/mocha@10.2.0/node_modules/mocha/lib/nodejs/esm-utils.js:100:20)
    at singleRun (/workspaces/FluidFramework/node_modules/.pnpm/mocha@10.2.0/node_modules/mocha/lib/cli/run-helpers.js:125:3)
    at Object.exports.handler (/workspaces/FluidFramework/node_modules/.pnpm/mocha@10.2.0/node_modules/mocha/lib/cli/run.js:370:5)
```

Not exporting it at all causes api-extractor to complain:

```console
@fluid-internal/client-utils: error during command 'api-extractor run --local' (exit code 1)
@fluid-internal/client-utils: api-extractor 7.38.3  - https://api-extractor.com/
@fluid-internal/client-utils: 
@fluid-internal/client-utils: Using configuration from ./api-extractor.json
@fluid-internal/client-utils: Analysis will use the bundled TypeScript version 5.1.6
@fluid-internal/client-utils: 
@fluid-internal/client-utils: API Extractor completed with errors
@fluid-internal/client-utils: 
@fluid-internal/client-utils: Warning: You have changed the public API signature for this project. Updating api-report/client-utils.api.md
@fluid-internal/client-utils: Error: src/bufferNode.ts:33:14 - (ae-forgotten-export) The symbol "Buffer_2" needs to be exported by the entry point index.d.ts
```

Exporting it as `type Buffer` seems to make everything happy.